Packs/Office365/ReleaseNotes/1_0_10.md

```diff
@@ -0,0 +1,6 @@
+
+#### Modeling Rules
+
+##### Office 365 Modeling Rule
+
+Updated the Modeling Rule logic for **msft_o365_general_raw**, casting the field *targetyammeruserid* as String for backend compatibility.
```

Comment:
Should this be "backend compatibility" or "backward compatibility"?

Reply:
backend